src/web-app.ts

```diff
@@ -23,11 +23,7 @@ webApp.container.singleton('Adonis/Core/Config', () => {
       }
     : {
         connection: {
-          host: knexfile.connection.host,
-          port: knexfile.connection.port,
-          user: knexfile.connection.user,
-          password: knexfile.connection.password,
-          database: knexfile.connection.database,
+          ...knexfile.connection
```

Comment:
I think would be better if you use it directly instead of spreading it.
`connection: knexfile.connection`